### PR TITLE
261 restrict write instruction

### PR DESCRIPTION
**Pull request by vakond**
* WriteHolder instruction is used instead of Write
* Holder accounts are created with seed generated with keccak_256 not shake_256

### CODE REVIEW

File: proxy/plugin/solana_rest_api_tools.py [hunk 1]
```diff
@@ -1168,6 +1168,81 @@ def write_trx_to_holder_account(signer, client, holder, ethTrx):
         logger.debug("confirmed: %s", rcpt)
 
 
+def deploy_contract(signer, client, ethTrx, perm_accs, steps):
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api_tools.py]: Why did you add this function? We drop it earlier and now this functionality locates in `call_signed`.

Comment (vakond) [reply proxy/plugin/solana_rest_api_tools.py]: The humor is I didn't add the function :). May be it's some merge quirk.

It still presents in some code of the develop branch though:
```
proxy-model.py $ sw develop
Already on 'develop'
Your branch is up to date with 'origin/develop'.
08:35:31 ~/projects/current/proxy-model.py $ fpy deploy_contract
     1	proxy/plugin/solana_rest_api_tools.py:43:location_bin = ".deploy_contract.bin"
     2	proxy/test_cancel_hanged.py:83:        cls.deploy_contract(cls)
     3	proxy/test_cancel_hanged.py:123:    def deploy_contract(self):
     4	proxy/test_eth_getLogs.py:65:        cls.deploy_contract(cls)
     5	proxy/test_eth_getLogs.py:74:    def deploy_contract(self):
```

I'm ready to remove the function `deploy_contract` from this branch.

Comment (vakond) [reply proxy/plugin/solana_rest_api_tools.py]: Removed the function:
```
def deploy_contract(signer, client, ethTrx, perm_accs, steps)
```

**Review by s-medvedev — APPROVED** [review]
